### PR TITLE
task: adds fonts

### PR DESCRIPTION
**Pull request by skibinska**
### This PR

- Adds fonts
- Adds `fonts.json`
- Adds a custom format and transformer to generate a font-face file called `fonts.css`.

### CODE REVIEW

File: utils/formats/formatFontFace.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/**
+ *
+ * @param allTokens
+ * @param options
+ * @returns {*}
+ *
+ * @see @see {@link https://github.com/amzn/style-dictionary/blob/main/examples/advanced/font-face-rules/sd.config.js#L18}
```
Comment (chriddell) [inline utils/formats/formatFontFace.js:7]: typo here (`@see @see`)

File: utils/formats/formatFontFace.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            `\n\tfont-style: ${style};`,
+            `\n\tfont-weight: ${weight};`,
+            `\n\tsrc: ${urls.join(',\n\t\t\t ')};`,
+            '\n}\n'
+        ].join('');
+
+        fontList.push(fontCss);
+
+        return fontList;
+
```
Comment (chriddell) [inline utils/formats/formatFontFace.js:45]: unnecessary new line here?

File: utils/formats/formatFontFace.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    const fontPathPrefix = options.fontPathPrefix || '../';
+
+    // https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/src
+    const formatsMap = {
+        'woff2': 'woff2',
+        'woff': 'woff',
+        'ttf': 'truetype',
+        'otf': 'opentype',
+        'svg': 'svg',
+        'eot': 'embedded-opentype'
```
Comment (chriddell) [inline utils/formats/formatFontFace.js:19]: do we actually need these formats? from my (limited!) reading, I don't think these are recommended/required any more...

https://stackoverflow.com/a/37091681

File: build.js [hunk 4]
```diff
@@ -67,12 +82,25 @@ async function run() {
   for (const cfg of configs) {
     const sd = new StyleDictionary(cfg);
 
+    sd.registerFormat({
+      name: 'custom/font-face',
+      format: ({ dictionary: { allTokens }, options }) => formatFontFace(allTokens, options)
+
```
Comment (chriddell) [inline build.js:88]: unnecessary new line here?

**Review by chriddell — APPROVED** [review]
All works for me locally! 

I find the detail of the code a bit hard to follow, but that's just because I've not used style dictionary before I think... a couple of thoughts I had, which might/probably already crossed your mind:

- any plans to convert some of this typescript?
- we'd probably benefit from some basic linting rules (I made a couple of comments about empty lines, which we could easily eliminate with eslint - would be happy to create a PR for this myself, but it can wait of course)

**Comment by skibinska** [comment]
Thank you, @chriddell! Yes, it would be really amazing if you could find some time for the eslint PR! 

I am happy to have a chat about how we can simplify the style dictionary setup. 

I can definitely take a look at adding TypeScript.

**Review by chriddell — APPROVED** [review]
